ecleankernel/layout/blspec.py

```diff
@@ -70,37 +79,70 @@ def find_kernels(self,
             exclusions=exclusions,
             module_directory=self.root / 'lib/modules')
 
-        # collect from /boot
         kernels: typing.Dict[str, Kernel] = {}
-        for ver in os.listdir(self.bootdir):
-            if ver.startswith('.'):
-                continue
-            dir_path = self.bootdir / ver
-            if dir_path.is_symlink() or not dir_path.is_dir():
-                continue
+        # collect from /boot
+        if self.bootdir.is_dir():
+            for ver in os.listdir(self.bootdir):
```

Comment:
Can we use Pathlib here?

Reply:
I'm not very familiar with Pathlib. Could you elaborate what you would specifically like to do with Pathlib? 

Note that this part of the code is unchanged, it has simply been prepended with an extra if statement and indented more.

Reply:
Don't stress about it for now, I just noticed the `/` separator & listdir call which felt a bit brittle. Let's leave it for another time.